distribution/docs/src/main/resources/_contents/_catalog-frameworks/standard-catalog-framework-contents.adoc

```diff
@@ -0,0 +1,152 @@
+
+==== Standard Catalog Framework
+
+The Standard Catalog Framework provides the reference implementation of a Catalog Framework that implements all requirements of the ${ddf-catalog} API. 
+`CatalogFrameworkImpl` is the implementation of the ${branding} Standard Catalog Framework.
+
+The Standard Catalog Framework is the core class of ${branding}.
+It provides the methods for query, create, update, delete, and resource retrieval (QCRUD) operations on the `Sources`.
+By contrast, the Fanout Catalog Framework only allows for query and resource retrieval operations, no catalog modifications, and all queries are enterprise-wide.
+
+Use this framework if:
+
+* access to a catalog provider to create, update, and delete catalog entries are required.
```

Comment:
I don't think it's _that_ pedantic actually. 
I went with 'access to a catalog provider is required to create, update, and delete catalog entries.'